src/osmf/js/osmf_flash.js

```diff
@@ -94,7 +94,8 @@ require("../../../html5-common/js/utils/constants.js");
     }
     this.encodings = testForFlash();
     this.technology = OO.VIDEO.TECHNOLOGY.FLASH;
-    this.features = [ OO.VIDEO.FEATURE.CLOSED_CAPTIONS ];
+    this.features = [ OO.VIDEO.FEATURE.CLOSED_CAPTIONS,
+                      OO.VIDEO.FEATURE.BITRATE_CONTROL ];
```

Comment:
the bitrate control feature is not supported unless the plugin allows the player to set a bitrate and i didn't see the logic for that below